openfisca_web_api/loader/variables.py

```diff
@@ -78,6 +82,9 @@ def build_variable(variable, country_package_metadata, tax_benefit_system):
             ),
         }
 
+    if variable.documentation:
```

Comment:
The way we handle unspecified documentation in the Web API is currently inconsistent:
- `documentation: null for parameters`
- `documentation: "" for formulas`
- no `documentation` key for variables

I'd be in favor of generalizing the 3rd option, as it makes the JSON more readable, lighter, and doesn't change anything for JS clients.

Reply:
Fixed for Web API : a documentation attribute is exposed only when it exists and its value is cleaned from start and end spaces.

> On Python API : the documentation attribute value is not modified at all.
For variable formulas, there is no specific `documentation` attribute as the user can already get  formula docstring with `formula.__doc__`.